source/query-bar.txt

```diff
@@ -57,7 +57,7 @@ operator.
    Builder to enter a query into the query bar. For details, see
    :ref:`build-query`.
 
-As you type, the :guilabel:`Apply` button is disabled and the
+As you type, the :guilabel:`Find` button is disabled and the
 :guilabel:`Filter` label turns red until a valid query is entered.
 Filter syntax requires strict :manual:`extended JSON
 </reference/mongodb-extended-json/>` types. For instance, to specify an
```

Comment:
Or, you could just have it say: "To specify an ``ObjectID``, use the format ``{ "$oid": "<string>" }``:"